modules/nw-infw-operator-rules-object.adoc

```diff
@@ -0,0 +1,155 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-node-firewall-operator.adoc
+
+:_content-type: CONCEPT
+[id="nw-ingress-node-firewall-operator-rules-object_{context}"]
+= Ingress Node Firewall rules object
+
+The fields for the Ingress Node Firewall rules object are described in the following table:
+
+.Ingress Node Firewall rules object
+[cols=".^2,.^2,.^6a",options="header"]
+|====
+|Field|Type|Description
+
+|`metadata.name`
+|`string`
+|The name of the CR object. The name of the config object must be `ingressnodefirewall`.
```

Comment:
@JoeAldinger The name of object does not need to be ingressnodefirewall because there can be more than one objects and this CR is for creating multiple firewall rules.